src/publishers/github.js

```diff
@@ -31,7 +31,8 @@ export default async (artifacts, packageJSON, forgeConfig, authToken, tag) => {
           repo: forgeConfig.github_repository.name,
           tag_name: tag || `v${packageJSON.version}`,
           name: tag || `v${packageJSON.version}`,
-          draft: true,
+          draft: forgeConfig.github_repository.draft !== false,
```

Comment:
I feel as though these should be read in from `process.argv` rather than config options in `config.forge`.  Mainly because I don't feel a change in publish target should require a commit into VC.

I.e. I would like these to be `electron-forge publish --target=github --flags=draft,prerelease` or something like that.

Just so it is decided at run time not at config time (if that makes sense).

@malept What do you think?